cmd/system-probe/api/client/client_windows.go

```diff
@@ -8,14 +8,38 @@
 package client
 
 import (
-	"net/http"
+	"context"
+	"fmt"
+	"net"
+	"time"
 
-	processNet "github.com/DataDog/datadog-agent/pkg/process/net"
+	"github.com/Microsoft/go-winio"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	idleConnTimeout = 5 * time.Second
```

Comment:
Any reason why the idle connect timeout is lower in windows?

Reply:
I'm not sure, I just preserved what it was using before. Perhaps @derekwbrown knows.